tests/utils/derivatives.c

```diff
@@ -2,10 +2,8 @@ void compute(double values[], int num_points, double ders[]){
 
 	for(int i = 0; i < num_points; ++i)
 	{
-		double k = values[i* 2 + 0 ];
-		double j = values[i* 2 + 1 ];
-		ders[i*2+0]= (((((((k * (((((k * (1) + k * (1))) + ((k * (0) + 3 * (1))))) - ((4 * 1 - k * 0)/ (4 * 4))) - (((k * k) + (3 * k)) - (k / 4)) * 1)/ (k * k)) + ((k * ((k * ((k * (1) + k * (1))) + (k * k) * (1))) + ((k * k) * k) * (1))))) + ((((22 / 7) * k) * ((k * (1) + k * (1))) + (k * k) * ((k * ((7 * 0 - 22 * 0)/ (7 * 7)) + (22 / 7) * (1))))))) + ((j * ((k * ((k * ((k * ((k * ((k * ((k * ((k * ((k * (1) + k * (1))) + (k * k) * (1))) + ((k * k) * k) * (1))) + (((k * k) * k) * k) * (1))) + ((((k * k) * k) * k) * k) * (1))) + (((((k * k) * k) * k) * k) * k) * (1))) + ((((((k * k) * k) * k) * k) * k) * k) * (1))) + (((((((k * k) * k) * k) * k) * k) * k) * k) * (1))) + ((((((((k * k) * k) * k) * k) * k) * k) * k) * k) * (0)))); // df/(k) 
-		ders[i*2+1]= (((((((k * (((((k * (0) + k * (0))) + ((k * (0) + 3 * (0))))) - ((4 * 0 - k * 0)/ (4 * 4))) - (((k * k) + (3 * k)) - (k / 4)) * 0)/ (k * k)) + ((k * ((k * ((k * (0) + k * (0))) + (k * k) * (0))) + ((k * k) * k) * (0))))) + ((((22 / 7) * k) * ((k * (0) + k * (0))) + (k * k) * ((k * ((7 * 0 - 22 * 0)/ (7 * 7)) + (22 / 7) * (0))))))) + ((j * ((k * ((k * ((k * ((k * ((k * ((k * ((k * ((k * (0) + k * (0))) + (k * k) * (0))) + ((k * k) * k) * (0))) + (((k * k) * k) * k) * (0))) + ((((k * k) * k) * k) * k) * (0))) + (((((k * k) * k) * k) * k) * k) * (0))) + ((((((k * k) * k) * k) * k) * k) * k) * (0))) + (((((((k * k) * k) * k) * k) * k) * k) * k) * (0))) + ((((((((k * k) * k) * k) * k) * k) * k) * k) * k) * (1)))); // df/(j) 
+		double T = values[i* 1 + 0 ];
```

Comment:
Is this file a part of results? In that case, I can delete it

Reply:
Or maybe we can add it to a gitignore. there are a number of these.

Reply:
I'll add this stuff to the .gitignore.

Reply:
Put all of the generated files into .gitignore.